### DISCUSSION TITLE
Any way to enable extended memory entitlements?

### DISCUSSION THREAD

**Discussion by o-gui-blindao** [utmapp/UTM#6591] [UTM Help (iOS)]
**Describe the issue**
UTM Crashes due to too much memory usage, usually when starting too many firefox tabs, OS is Debian 12 arm64 maybe enabling extended memory entitlements could help?
**Configuration (required)**
* UTM Version: 4.5.3
* OS Version: iPadOS 15.4
* Device Model: iPad Air 5 (M1)
* Is it jailbroken (name jailbreak used)? Trollstore and dopamine installed
* How did you install UTM? Trollstore, might install via jailbreak once karen’s repo isn’t down (appsync unified required)

**Crash log**
UTM didn’t drop any crashlog for some reason, probably because it got closed by iOS

**Debug log**
[debug.log](https://github.com/user-attachments/files/16114632/debug.log)

**Comment by osy** [comment]
If you are using trollstore, it is already holding the extended memory entitlement. What you're experiencing is just out of memory.
